pkg/zpdb/config_observer.go

```diff
@@ -84,16 +86,20 @@ func (c *configObserver) addOrUpdate(obj *unstructured.Unstructured) {
 	updated, generation, err := c.pdbCache.addOrUpdateRaw(obj)
 	if err != nil {
 		level.Error(c.logger).Log("msg", "zpdb configuration error", "name", obj.GetName(), "err", err)
+		c.metrics.ConfigurationsObserved.WithLabelValues("invalid", obj.GetName()).Inc()
 	} else if updated {
 		level.Info(c.logger).Log("msg", "zpdb configuration updated", "name", obj.GetName(), "generation", generation)
+		c.metrics.ConfigurationsObserved.WithLabelValues("updated", obj.GetName()).Inc()
 	} else {
 		level.Info(c.logger).Log("msg", "zpdb configuration update ignored", "name", obj.GetName(), "generation", generation, "ignored-generation", obj.GetGeneration())
+		c.metrics.ConfigurationsObserved.WithLabelValues("ignored", obj.GetName()).Inc()
 	}
 }
 
 func (c *configObserver) onPdbAdded(obj interface{}) {
 	unstructuredObj, isUnstructured := obj.(*unstructured.Unstructured)
 	if !isUnstructured {
+		c.metrics.ConfigurationsObserved.WithLabelValues("not-unstructured", "unknown").Inc()
```

Comment:
Given we don't expect to receive values that aren't `Unstructured` instances, what if we group these together with `invalid` rather than introducing a special label value for it?

(same applies below)